README.md

```diff
@@ -31,26 +26,32 @@ join the discussion in [#498](https://github.com/sbt/sbt-native-packager/issues/
   Finally, any bugs or features you find you need, please report to our [issue tracker](https://github.com/sbt/sbt-native-packager/issues/new).
   Please check the [compatibility matrix](https://github.com/sbt/sbt-native-packager/wiki/Tested-On) to see if your system is able to
   produce the packages you want.
-  
-## Features
 
-* Build native packages for different systems
+## Features ##
+
+* Build [native packages][formats] for different systems
   * Universal `zip`,`tar.gz`, `xz` archives
   * `deb` and `rpm` packages for Debian/RHEL based systems
   * `dmg` for OSX
   * `msi` for Windows
   * `docker` images
 * Provide archetypes for common use cases
- * Java application with startscripts for linux/osx/windows
- * Java server additional autostart configurations
+ * [Java application] with startscripts for linux/osx/windows
+ * [Java server application][] with additional autostart configurations
```

Comment:
I'm not sure how this link format is supposed to work, but it seems to go to the same link as "Java application" above.